modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/continuous/CacheContinuousQueryHandler.java

```diff
@@ -913,7 +925,7 @@ private void onEntryUpdate(CacheContinuousQueryEvent evt, boolean notify, boolea
                     notifyLocalListener(evts, trans);
 
                     if (!internal && !skipPrimaryCheck)
-                        sendBackupAcknowledge(ackBuf.onAcknowledged(entry), routineId, ctx);
+                        sendBackupAcknowledge(ackBufBackup.onAcknowledged(entry), routineId, ctx);
```

Comment:
Why do not we call `cleanupBuffers` here? Should we cleanup buffer here? If yes it worth adding a test for that case.

Reply:
We shouldn't clear the buffer for local deployed listener since we're not using "entryBufs" inside "handleEntry" for this case. I have added additional test for this scenario: "testMultipleNodesWithBackupsWithoutClient".